src/Containers/TestScreen.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react'
```

Comment:
I've create this Screen to test our Redux integration, everytime the Update State button is clicked we're dispatching an action to increment a counter.